include/index.h

```diff
@@ -365,6 +365,13 @@ namespace diskann {
     DISKANN_DLLEXPORT size_t load_tags(const std::string tag_file_name);
     DISKANN_DLLEXPORT size_t load_delete_set(const std::string &filename);
 
+    void select_starting_points(
+        Parameters &parameters,
+        std::mt19937& gen,
```

Comment:
Is there a strong reason to pass the random generator? Can this not be instantiated inside the function?

Reply:
changed it to be inside